api/src/gotEnzymes/gene.js

```diff
@@ -8,7 +8,8 @@ const getGene = async value => {
   `;
 
   if (genes.length !== 1) {
-    throw new Error(`Gene ${value} returned ${gene.length} results.`);
+    console.log(`Gene ${value} returned ${genes.length} results.`);
```

Comment:
Shouldn't `console.log` be `console.error`?

Reply:
Yes, you're right. Fixed in 3fc0eaf.